services/ui-src/src/components/cards/TemplateCard.tsx

```diff
@@ -78,7 +78,7 @@ export const TemplateCard = ({
 };
 
 interface Props {
-  templateName: string;
+  templatename: string;
```

Comment:
I'm not sure I understand this change 🤔 
what does having `templatename` in the DOM do? Also I compared main and this branch and it's all lowercase in both DOMs

Reply:
I can provide some more context! :) 

This change was specifically to fix this error: 

```
Warning: React does not recognize the `templatename` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `templatename` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

```
> Also I compared main and this branch and it's all lowercase in both DOMs

Oh hmmm, that is curious. Did it just get added anyways?

Reply:
Aha so I just tested in the deployed dev instance (https://mdctmfpdev.cms.gov) and compared to local instance on main and the error only shows up when running locally 🤔 I did not see it in the deployed environment.

Reply:
So `templateName` is passed in from HomePage but it's never actually used in TemplateCard. Could we remove that prop and then we won't have this error or the extra code? I think right now the attribute is getting spread into the Flex via ...props, but I don't think we need it in the DOM anyway. It seems like verbiage handles all of the differences between each template card type